dosagelib/plugins/old.py

```diff
@@ -635,10 +635,12 @@ def getmodules(cls):
             # Renamed in 2.16
             cls('1997', '1977'),
             cls('ApartmentForTwo', 'NamirDeiter/ApartmentForTwo'),
-            cls('Catena', 'CatenaManor/CatenaCafe'),
+            cls('CaribbeanBlue', 'KemonoCafe/CaribbeanBlue'),
```

Comment:
Even if that seems a bit arbitrary (and that was a loong time ago because I was being a bad maintainer), I only wanted to add comics that were in the previous release to this list - so this one (`CaribbeanBlue`) doesn't qualify.